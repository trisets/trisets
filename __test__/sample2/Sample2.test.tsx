import Sample2 from '@/pages/sample2';
import { render } from '@testing-library/react';
import { expect, test } from 'vitest';

test('sample2', () => {
  const { container } = render(<Sample2 />);
  expect(container.firstChild).toMatchSnapshot();
});
