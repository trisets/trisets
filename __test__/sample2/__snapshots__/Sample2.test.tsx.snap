// Vitest Snapshot v1

exports[`sample2 1`] = `
<h1>
  Page02 nandesu
</h1>
`;
